### PR TITLE
🍱 New logo supports dark mode and fits better within README

### PR DESCRIPTION
**Pull request by brds**
# Current

Current logo on README doesn't support dark mode:

image

# Proposition

Updated logo where text features an outline so it's still readable in dark mode:

image
image

<p align="center"><i>(Click to enlarge)</i></p>

### Also

Other changes are included:
- logo is smaller and re-organised so it fits better on the README header. I assumed this logo is intended to be used only in this context
- I removed the baseline "Property-based testing" because it was redundant with the next sentence in the file.
- previous font was (I think) FF Meta, which is not a free font. I took the initiative to replace it with Source Sans Pro. It looks alike and is available as a Google font, giving you more control and autonomy on further updates.
- I removed the monkey's nostrils, I think he looks funnier like this :¬)

## In a nutshell

❌ New feature
❌ Fix an issue
❌ Documentation improvement
✔️ Other: logo on README update, see above

(✔️: yes, ❌: no)

## Potential impacts

The only impact is what the README will look like

### CODE REVIEW

**Comment by brds** [comment]
@dubzzz PR is ready for review

**Comment by dubzzz** [comment]
Awesome 🎉

**Comment by dubzzz** [comment]
We should also update https://www.github.com/dubzzz/fast-check/tree/main/logo%2Fmug.svg
